games/static/games/proto.js

```diff
@@ -1,14 +1,48 @@
+class Game{
+    constructor() {
+
+    }
+
+}
+
+class Setup{
+    constructor(alienLocations, gameSpeed, movementPattern) {
+        this.alienLocations = alienLocations
+        this.gameSpeed = gameSpeed
+        this.movementPattern = movementPattern
+    }
+    getAlienLocations() {
+        return this.alienLocations
+    }
+    getGameSpeed() {
+        return this.gameSpeed
+    }
+    getMovementPattern() {
+        return this.movementPattern
+    }
+}
+
 class Square{
-    constructor(indexOnBoard, div) {
+    constructor(indexOnBoard, div, lineWidth) {
         this.index = indexOnBoard
         this.div = div
+        this.lineWidth = lineWidth
     }
     setCharacter(character) {
         this.character = character
         let cssClass = character.getCssClass()
         this.div.classList.add(cssClass)
```

Comment:
i think you wanna override it right? this will add more and more classes as we set more charechters

Reply:
what do you mean? i dont understand...

Reply:
i got it reset the class not just add